.github/workflows/deploy.yml

```diff
@@ -29,64 +29,37 @@ on:
         required: true
       DEPLOY_PRIVATE_KEY:
         required: true
+      DOCKER_REGISTRY_ADDRESS:
+        required: true
+      DOCKER_REGISTRY_NAMESPACE:
+        required: true
+      DOCKER_REGISTRY_USERNAME:
+        required: true
+      DOCKER_REGISTRY_PASSWORD:
+        required: true
 
 jobs:
   deploy:
     runs-on: ubuntu-latest
     environment: ${{ inputs.build-env }}
-
     steps:
-      - uses: actions/checkout@v2.4.0
-
-      - name: Use Node.js 16.14.0
-        uses: actions/setup-node@v2.5.1
-        with:
-          node-version: 16.14.0
-
-      - name: Resolve prod dependencies
-        id: prod-dep-caches
-        uses: actions/cache@v2
-        with:
-          path: |
-            node_modules
-            packages/*/node_modules
-          key: ${{ runner.os }}-production-${{ hashFiles('package-lock.json', 'packages/*/package-lock.json') }}
-          restore-keys: ${{ runner.os }}-production
-
-      - uses: actions/download-artifact@v2
-        name: Download ${{ inputs.build-env }} artifact
-        with:
-          name: target-${{ inputs.build-env }}
-
-      - name: Un-compress target
-        run: tar -zxf target.tar && rm target.tar
-
-      - name: Dump envs
```

Comment:
we use `.env.local` here instead of `.env`, please move these codes before building docker image.

Reply:
OK